python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1451,6 +1452,12 @@ def get_asset_records(
     ) -> Iterable["AssetRecord"]:
         return self._event_storage.get_asset_records(asset_keys)
 
+    @traced
+    def get_all_event_tags_for_asset(
+        self, asset_key: AssetKey, key: Optional[str] = None
```

Comment:
maybe rename to be `tag_key`?

and sorry for the back and forth, but maybe if we're applying a filter, it's confusing to have `get_all_` instead of just `get_`